app/views/neighborhoods/new.html.erb

```diff
@@ -1,3 +1,11 @@
+<% if @neighborhood.errors.present? %>
+  <ul>
+    <% @neighborhood.errors.full_messages.each do |error| %>
+      <li><%= error %></li>
+    <% end %>
+  </ul>
+<% end %>
+
```

Comment:
What would happen if you put these error bits in the layout, rather than in multiple views? Or else extracted to a partial?